solutions/src/main/java/ru/sberbank/school/task08/JAXBManager.java

```diff
@@ -0,0 +1,78 @@
+package ru.sberbank.school.task08;
+
+import lombok.NonNull;
+import ru.sberbank.school.task08.state.*;
+import ru.sberbank.school.util.Solution;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import javax.xml.bind.Unmarshaller;
+import java.io.*;
+import java.util.List;
+import java.util.Objects;
+
+@Solution(8)
+public class JAXBManager extends SaveGameManager<JAXBMapState<GameObject>, GameObject> {
```

Comment:
Abbreviation in name &#39;JAXBManager&#39; must contain no more than &#39;2&#39; consecutive capital letters.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1720282771>*